hackathon_site/registration/test_admin.py

```diff
@@ -0,0 +1,62 @@
+import csv
+import io
+
+from django.test import TestCase
+from django.urls import reverse
+from rest_framework import status
+
+from hackathon_site.tests import SetupUserMixin
+from registration.admin import ApplicationAdmin
+
+
+class ExportCsvAdminTestCase(SetupUserMixin, TestCase):
+    """
+    Test that CSV exportation in the Registration/Application tab works as expected.
+    """
+
+    def setUp(self):
+        super().setUp()
+        self.application_view = reverse("admin:registration_application_changelist")
+        self.export_fields = [field[0] for field in ApplicationAdmin.export_fields]
+
+        self.user.is_staff = True
+        self.user.save()
+
+        self.team1 = self._make_full_registration_team(self_users=False)
+        self.team2 = self._make_full_registration_team(self_users=False)
+
+        self.data = {
+            "action": "export_as_csv",
+            "_selected_action": [i.id for i in self.team1.applications.all()],
+        }
+
+    def test_permission_denied(self):
+        self._login()
+
+        response = self.client.post(self.application_view, self.data)
+        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+
+    def test_returns_csv_successfully(self):
```

Comment:
Could this test be made a bit more thorough? Currently it's testing just headers and emails. At the very least, it would be good to test the name and review status fields since they use the `rgetattr` function which is untested for nested attributes currently